README.adoc

```diff
@@ -0,0 +1,114 @@
+= Agriboard
+
+Agriboard is an administrative dashboard for the agriculture software stack.
+It is used to manage users and view advanced analytics.
+
+== Running Using Docker
+
+The repository can be run with docker and not installing any other dependencies.
+The `dockerize.sh` script has been made for this purpose.
+The `./dockerize.sh` script can be used with the following arguments:
+
+* `init` - creates a Docker image based on the Dockerfile
+* `build` - installs the node modules and build the project using `npm run build`
+* `run` - installs the node modules and starts a development web server at `<docker_ip>:8080`
+== Dependencies
+
+. link:https://nodejs.org/en/[NodeJS] (version 8.9.0 or higher) is the base framework
+
+. link:https://yarnpkg.com/lang/en/docs/install/[Yarn] installs/manages all other dependencies
+
+To install all other dependencies, run `yarn`
+
+IMPORTANT: Please use yarn to manage your dependencies.
+
+=== Using Yarn
+
+The following commands should never be used in this repository.

```

Comment:
Maybe bold this so it's clear these are commands nobody should be running. I'm worried about skimmers that will just see the commands and instinctively run them :P

If you don't think this is necessary I'll adhere to your best judgement :+1:

Reply:
I agree, I'll do that. Won't those commands fail without a `package-lock.json` file?

Reply:
No they shouldn't fail I don't think, and they will probably even generate a `package-lock.json` file (I don't know much about the file in general I didn't know until this project npm implemented a lock file themselves)